secure_redis/settings.py

```diff
@@ -4,12 +4,19 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
-def get_secure_cache_opts():
+def get_secure_cache_opts(require_options=False):
```

Comment:
I tried it now, since I added `secure_redis` as an app inside `base.py` then caused `__init__.py` in `secure_redis` to be loaded. So if we set the `require_options` to be True, this will raise the `ImproperlyConfigured` below.

But I was thinking that we don't need to set `secure_redis` in the `INSTALLED_APPS`, I remember I added it previously since we were doing data migration using migration file, but since we removed the migration file, there is no need to it.

So based on this, I will remove the `secure_redis` from installed apps, then remove this flag `require_options` and make it always required.